src/mbgl/renderer/image_manager_observer.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+namespace mbgl {
+
+class ImageManagerObserver {
+public:
+    virtual ~ImageManagerObserver() = default;
```

Comment:
nit: observer destructor is better to be protected, as the observable object does not own the observer instances (i.e. does not delete them).